pymeasure/instruments/keithley/keithley2200.py

```diff
@@ -31,85 +31,92 @@
 
 
 class PSChannel(Channel):
-    """ Implementation of a Keithley 2200 channel. """
+    """Implementation of a Keithley 2200 channel."""
 
     VOLTAGE_RANGE = [0, 70]
     CURRENT_RANGE = [0, 45]
 
     output_enabled = Instrument.control(
-        "SOURCE:OUTP:ENAB?", "SOURCE:OUTP:ENAB %d",
-        """ A boolean property representing output state.""",
+        "SOURCE:OUTP:ENAB?",
+        "SOURCE:OUTP:ENAB %d",
+        """ Enables or Disables the output.""",
         validator=strict_discrete_set,
         values={True: 1, False: 0},
-        map_values=True
+        map_values=True,
     )
 
-    voltage = Instrument.control(
-        "VOLT?", "VOLT %g",
-        """ A floating point property representing the output voltage in Volts.""",
+    voltage_setpoint = Instrument.control(
+        "VOLT?",
+        "VOLT %g",
+        """ Set output voltage in Volts.""",
```

Comment:
You can read the output setpoint as well, therefore "control":
```suggestion
        """ Control output voltage in Volts.""",
```